### DISCUSSION TITLE
Branch automerge isn't running despite checks passing

### DISCUSSION THREAD

**Discussion by bigolu** [renovatebot/renovate#32093] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

The dashboard says that one of my branch-automerge updates is awaiting pending status checks, but they've all completed. The checks either completed successfully or were skipped.

To see if the skipped checks were the issue, I tried opening a pull request for the branch just to see if GitHub would allow me to merge and it did. I didn't actually merge it though and I've since closed the pull request.

I checked the logs and saw that there were "No check runs found" so I'm thinking that's why the update is stuck in pending. I used the GitHub CLI to get the checks myself, using what I believe is the same endpoint that Renovate uses, and I got 3 checks back. The command I ran was:

```bash
gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" '/repos/bigolu/system-configurations/commits/renovate/branch-automerge/lycheeverse-lychee-action-digest/check-runs'
```

Branch with update: [renovate/branch-automerge/lycheeverse-lychee-action-digest](https://github.com/bigolu/system-configurations/tree/renovate/branch-automerge/lycheeverse-lychee-action-digest)

### Logs (if relevant)

<details><summary>Logs</summary>

```

DEBUG: Branch automerge is enabled (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
DEBUG: resolveBranchStatus(branchName=renovate/branch-automerge/lycheeverse-lychee-action-digest, ignoreTests=false) (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
DEBUG: getBranchStatus(renovate/branch-automerge/lycheeverse-lychee-action-digest) (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
DEBUG: http cache: saving https://api.github.com/repos/bigolu/system-configurations/commits/renovate/branch-automerge/lycheeverse-lychee-action-digest/status (etag=W/"147fbfe51904262ea5318dde1f4ca3ffd348c60e8379bf18caa2ebe90477c4c3", lastModified=undefined) (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
DEBUG: branch status check result (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
{
  "state": "pending"
  "statuses": []
}

DEBUG: Successful checks are all internal renovate/ checks, so returning "pending" branch status (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
DEBUG: No check runs found (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")
{
  "result": {
    "total_count": 0,
    "check_runs": []
  }
}

DEBUG: Branch status yellow (branch="renovate/branch-automerge/lycheeverse-lychee-action-digest")

```

</details>

**Comment by rarkins** [comment, marked as answer]
Are you sure that renovate has run again since those checks completed?

**Reply by bigolu** [reply]
Oh right I've been rebasing as opposed to triggering a scan and so the checks are being run again. Thanks.
